src/App.js

```diff
@@ -23,6 +40,51 @@ export class App {
     this.clippingState = new Set();
     this.mutedState = new Set();
     this.isAudioPaused = new Set();
+
+    this.world = createWorld();
+
+    // TODO: Create accessor / update methods for these maps / set
+    this.world.eid2obj = new Map(); // eid -> Object3D
+
+    // HACK: We are setting the network id of the world to zero
+    this.world.nid2eid = new Map();
```

Comment:
I want to think up a better pattern for storing system state like this. Here we are putting it on world, in other spots like camera tool we just store it in a module local variable. I think having some pattern for this would be useful.